tests/test_modalities/test_ecephys/test_tools_spikeinterface.py

```diff
@@ -1007,6 +1007,33 @@ def test_missing_bool_values(self):
         assert np.array_equal(extracted_complete_property, expected_complete_property)
         assert np.array_equal(extracted_incomplete_property, expected_incomplete_property)
 
+    def test_spikeinterface_physical_unit_properties_excluded(self):
+        """Test that new SpikeInterface physical unit properties are excluded from electrodes table."""
+        # Create a recording with the new SpikeInterface properties that should be excluded
+        recording = generate_recording(num_channels=4, durations=[1.0])
+        recording.set_property("gain_to_physical_unit", [1.0, 1.5, 2.0, 2.5])
+        recording.set_property("offset_to_physical_unit", [0.0, 0.1, 0.2, 0.3])
+        recording.set_property("physical_unit", ["uV", "mV", "uV", "mV"])
+        
+        # Add electrodes to NWBFile
+        add_electrodes_to_nwbfile(recording=recording, nwbfile=self.nwbfile)
+        
+        # Verify that the new properties are NOT in the electrodes table
+        electrodes_columns = list(self.nwbfile.electrodes.colnames)
+        excluded_properties = ["gain_to_physical_unit", "offset_to_physical_unit", "physical_unit"]
+        
+        for prop in excluded_properties:
+            self.assertNotIn(
+                prop,
+                electrodes_columns,
+                f"Property '{prop}' should be excluded from electrodes table as it belongs in ElectricalSeries"
+            )
+        
+        # Verify that standard properties are still included
+        expected_columns = ["location", "group", "group_name", "channel_name"]
+        for col in expected_columns:
+            self.assertIn(col, electrodes_columns, f"Standard column '{col}' should be present")
+
```

Comment:
This test is still failing pre-commit validation. Please run pre-commit on this file to fix the the formatting issue(s).